src/packages/cli/src/cli.ts

```diff
@@ -54,6 +55,17 @@ if (argv.action === "start") {
   const flavor = argv.flavor;
   const cliSettings = argv.server;
 
+  const loggingOptions = (argv as any).logging;
+  let closeLogHandle = null;
+  if (loggingOptions?.file) {
+    const { log, close } = createLogger({
+      baseLog: console.log,
+      file: loggingOptions.file
+    });
+    loggingOptions.logger = { log };
+    closeLogHandle = close;
+  }
+
```

Comment:
This is clearly terrible, as the logging configuration is, untyped. The logging config is currently flavor specific - I think we should pull it up to be a "top level" Ganache config similar to Server config. I think we should do that in a separate PR though.

Thoughts?

Reply:
I have no opinion, but curious about your reasoning to defer to follow on work.